Makefile

```diff
@@ -56,15 +60,16 @@ all: build test
 build: LDFLAGS += -X github.com/cockroachdb/cockroach/util.buildTag "$(shell git describe --dirty)"
 build: LDFLAGS += -X github.com/cockroachdb/cockroach/util.buildTime "$(shell date -u '+%Y/%m/%d %H:%M:%S')"
 build: LDFLAGS += -X github.com/cockroachdb/cockroach/util.buildDeps "$(shell GOPATH=${GOPATH} build/depvers.sh)"
+build: TAGS += release
 build:
-	$(GO) build $(GOFLAGS) -ldflags '$(LDFLAGS)' -v -i -o cockroach
+	$(GO) build -tags '$(TAGS)' $(GOFLAGS) -ldflags '$(LDFLAGS)' -v -i -o cockroach
```

Comment:
Eventually we're trying to have as few outside dependencies as possible, and the release binary will likely always have the netgo tag. Since Go1.4 doesn't rebuild the standard library, so if you run one-off go invocations without that tag, it'll just do nothing to what's there. The more important change is the `install_suffix=netgo`: That actually makes sure that there's always a netgo tag active when you use the Makefile. If you run `go` from your command like, you get install_suffix="", so go will just use whatever it finds in the standard install suffix. So if you start from a clean slate, run `make build` and then try to use `go` directly, yeah, you'll build it twice. But only on the first invocation.
I didn't see any downsides of this, and the upside is that we're all using netgo when using the Makefile. Seemed like a good idea.